pages/common/difft.md

```diff
@@ -0,0 +1,33 @@
+# difft
+
+> Compare files or directories based on the syntax of the programming language.
+> See also: `delta`, `diff`.
+> More information: <https://difftastic.wilfred.me.uk/introduction.html>.
+
+- Compare two files or directories:
+
+`difft {{path/to/original_file_or_directory}} {{path/to/new_file_or_directory}}`
```

Comment:
```suggestion
`difft {{path/to/source_file_or_directory}} {{path/to/target_file_or_directory}}`
```

Reply:
Source and target makes sense when transferring or copying files from one location to another. For diff, I don't think it is ideal.

Reply:
Done, used `path/to/file_or_directory1` and `path/to/file_or_directory2`